commons/connector/health/health_mergingExample2.avsc

```diff
@@ -0,0 +1,48 @@
+{
+  "namespace": "org.radarcns.active.healthkit",
+  "name": "HealthKit_Height_BodyMass_BodyMassIndex",
```

Comment:
My bad, didn't see this. Yes it should be like this, but without the duplicate `time` and `timeReceived` fields.